apps/fz_http/lib/fz_http_web/views/json/encoders/openid_connect_provider.ex

```diff
@@ -0,0 +1,5 @@
+defimpl Jason.Encoder, for: FzHttp.Configurations.Configuration.OpenIDConnectProvider do
```

Comment:
You can just add the @derive attribute to the module or, even better, define a view for this schema and use `reader('open_id_connect_provider.json, assigns)`.

Reply:
Good point. I like having another view for this.

Reply:
Fixed